### DISCUSSION TITLE
Comparing payer-side transparency files with provider-side

### DISCUSSION THREAD

**Discussion by iseff** [CMSgov/price-transparency-guide#413] [Q&A]
Perhaps this is a silly question, but...

We do a lot of work with provider side transparency files, and we're excited to compare those with these payer-side files.

One obvious thing I can't seem to figure out is why each line in the payer-side only appears to have one code/code type. Whereas in the provider side, most items will have a procedure code AND revenue code, for instance.

I'm sure I'm missing something here, but any help would be appreciated in order to help compare these files.

Thanks.

**Comment by iseff** [comment]
Ping! Any thoughts on this?

**Comment by rakeshkatte** [comment]
Claims I have seen mostly fall into the 2 categories (I wouldn't say I have
seen them all, so there might be other scenarios too):
1. If room and board days are on the claim, the pricing is done based on
these instead of the cpt codes.
2. If non room and board rev codes are on the claim, then the pricing is
done based on the cpt codes.

But we have not seen examples where the combination gives a different
pricing.
That being said, the current schema would work.

However, we do have scenarios where we would need a combination of ICD diag
codes and cpts, but now we are having to add one of these codes into the
generic "additional information" node to explain the pricing being
published.

**Reply by iseff** [reply]
Thanks, @rakeshkatte. On the provider side (hospital price transparency files), we see plenty of different prices based on revenue code and procedure code (CPT) combinations. So, interesting to see that you haven't noticed this. 

In either case, how would the current schema work exactly? I'm trying to figure that piece out still. Thanks again for the help!
